helper/mount/mount.go

```diff
@@ -167,6 +167,11 @@ func (h *Mount) Underlying() billy.Basic {
 	return h.underlying
 }
 
+// Capabilities implements the Capable interface.
+func (fs *Mount) Capabilities() billy.Capability {
+	return billy.Capabilities(fs.underlying)
```

Comment:
Why `fs.underlying` and not `fs.source`? If `Capabilities` are global to the filesystem, this should probably return `fs.underlying.Capabilities()&fs.source.Capabilities()`, which is the common denominator, even if there is some directory with additional capabilities.